Pod/Classes/Lifecycle/Behaviors/Nav-Bar-Hairline-Fade/NavBarHairlineFadeBehavior.swift

```diff
@@ -45,13 +45,20 @@ public extension NavBarHairlineFadeBehavior {
         set { navBarHairlineFadeUpdater.hairline.thickness = newValue }
         get { return navBarHairlineFadeUpdater.hairline.thickness }
     }
+    
+    var logsContentOffset: Bool {
```

Comment:
#req comment explaining what this does. There should be comments on the others too, but especially on this one because it's got a specific use for debugging.